<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="loginScreen">
     <widget class="QLabel" name="L_welcome">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>140</y>
        <width>531</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Welcome to Nong-Bong ATM!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="B_register">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>500</y>
        <width>90</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
     <widget class="QLabel" name="L_name">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>210</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="LE_password">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>260</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="inputMethodHints">
       <set>Qt::InputMethodHint::ImhHiddenText</set>
      </property>
     </widget>
     <widget class="QPushButton" name="B_enter">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>320</y>
        <width>90</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
     <widget class="QLabel" name="L_password">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>260</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="LE_name">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>210</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="L_noAccount">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>460</y>
        <width>801</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>No account?</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <zorder>L_welcome</zorder>
     <zorder>B_register</zorder>
     <zorder>L_name</zorder>
     <zorder>B_enter</zorder>
     <zorder>L_password</zorder>
     <zorder>LE_name</zorder>
     <zorder>LE_password</zorder>
     <zorder>L_noAccount</zorder>
    </widget>
    <widget class="QWidget" name="dashboardScreen">
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>180</y>
        <width>160</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="L_welcomeUser">
         <property name="text">
          <string>Welcome, user!</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="B_logout">
         <property name="text">
          <string>End work</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
